lib/Onnxifi/onnxifiGlow.cpp

```diff
@@ -18,6 +18,11 @@
 
 #include "glow/Importer/ONNXIFILoader.h"
 
+/// Allow defining names for onnxifi implementation.
+#ifndef ONNXIFI_LIBRARY_FUNCTION_WRAPPER
```

Comment:
I'd suggest to use a Glow-specific name, e.g. `GLOW_ONNXIFI_LIBRARY_FUNCTION_WRAPPER` to avoid name conflicts